core/mri.c

```diff
@@ -329,7 +329,6 @@ void GdbCommandHandlingLoop(void)
     } while (!startDebuggeeUpAgain);
 }
 
-__attribute__((weak)) uint32_t  mriPlatform_HandleGDBCommand(Buffer* pBuffer);
```

Comment:
I think we should revert this part of the change. There is no reason that the weak reference for this function should be known outside of this module.

Reply:
Yes, let's keep the prototype, but remove the weak attribute.